xml/obs_build_config.xml

```diff
@@ -158,7 +158,7 @@ Keep: debbuild</screen>
       <varlistentry>
        <term><parameter>vmfstype:</parameter></term>
        <listitem>
-        <para>Defines a specific file system when building inside of a VM. </para>
+        <para>Defines a specific file system when building inside of a VM. Possible values are iext2, ext3 ext4, btrfs, xfs, reiserfs (v3). There is also an option to disable dir indexing for the ext file systems. This has an effect to the performance, but would make the file ordering inside of directories reproducible. specify for example vmsftype:ext3:nodirindex for this.</para>
```

Comment:
This line is really really long. I have also tried to clarify a little. Also, is `iext2` correct or does it need to be `ext2` (_not_ corrected below)?

```suggestion
        <para>
         Defines a specific file system when building inside of a VM.
         Possible values are
         <literal>iext2</literal>, <literal>ext3</literal>,
         <literal>ext4</literal>, <literal>btrfs</literal>,
         <literal>xfs</literal>, <literal>reiserfs</literal> (v3).
         There is also an option to disable directory indexing for ext2, ext3,
         and ext4 file systems.
         This makes file ordering inside of directories reproducible but may
         have a negative performance impact.
         To apply this, specify
         <literal>vmsftype:ext3:nodirindex</literal> (replace
         <replaceable>ext3</replaceable> with the appropriate value for the
         file system).
        </para>
```